xla/service/buffer_assignment.cc

```diff
@@ -2332,7 +2347,8 @@ BufferAssigner::CreateAssignment(
     private_stack_colors.insert(color);
   }
 
-  assignment->CombineTempAllocations(private_stack_colors, temp_buffer_color);
+  std::ignore = assignment->CombineTempAllocations(private_stack_colors,
```

Comment:
Same here, we should preserve the previous behavior and CHECK fail if the returned status here is not ok.